mlir/lib/Bindings/Python/IRAttributes.cpp

```diff
@@ -72,6 +72,23 @@ or 255), then a splat will be created.
     type or if the buffer does not meet expectations.
 )";
 
+static const char kDenseElementsAttrGetFromListDocstring[] =
+    R"(Gets a DenseElementsAttr from a Python list of attributes.
```

Comment:
nit: maybe put a small note here about "this is expensive YMMV"

Reply:
Added and suggested that using a buffer instead might be faster